gen_breaktest.go

```diff
@@ -0,0 +1,215 @@
+// +build generate
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"go/format"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+//go:generate go run gen_breaktest.go
+
+// See https://www.unicode.org/license.html for the Unicode license agreement.
+
+// We want to test against a specific version rather than the latest, which
+// can be found at:
+// https://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakTest.txt
+// When/if the package is upgraded to a new version, change these to generate new tests.
+const (
+	url      = `https://www.unicode.org/Public/13.0.0/ucd/auxiliary/GraphemeBreakTest.txt`
+	filename = `GraphemeBreakTest-13.0.0.txt`
+)
+
+func main() {
+	log.SetPrefix("gen_breaktest: ")
+	log.SetFlags(0)
+
+	// Read text of testcases and parse into Go source code.
+	src, err := readAndParse()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Format the Go code.
+	srcfmt, err := format.Source(src)
+	if err != nil {
+		log.Fatalln("gofmt:", err)
+		//srcfmt = src
+	}
+
+	// Write it out.
+	if err := ioutil.WriteFile("grapheme_break_test.go", srcfmt, 0644); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// readAndParse reads a GraphemeBreakTest text file,
```

Comment:
I usually format my function comments to fit 80 characters per line. For consistency, it would be great if you could do that, too.

Reply:
I usually let it format to whatever my editor width currently is (usually 90 for Go source) but for shortish bits I have the habit of using par's `-l` option to "make the last line […] about the same length as the others" which I find makes it look nice for most  any width a user may see it as.

Just to confirm, you'd prefer a strict 80 column wrapping like so:

    // readAndParse reads a GraphemeBreakTest text file, either from a local file or
    // from a URL.

I can easily make all the comments do this via my editor as long as I know the parameters you'd like (e.g. 80 or 78, etc).